Automatic Validation ended with:
> 2024-01-18T03:56:12.0370106Z ##[error] Manifest Error: DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index. Existing DisplayVersion range in index:  [ [2023.1, 2023.1]]

(Automated response - build 604.)

--- a/manifests/g/Google/AndroidStudio/2023.1.1.27/Google.AndroidStudio.installer.yaml
+++ b/manifests/g/Google/AndroidStudio/2023.1.1.27/Google.AndroidStudio.installer.yaml
@@ -12,7 +12,7 @@ InstallerSuccessCodes:
 - 1223
 UpgradeBehavior: install
 AppsAndFeaturesEntries:
-- DisplayVersion: "2023.1.1"
+- DisplayVersion: "2023.1.1.27"
   ProductCode: Android Studio
 Installers:
 - Architecture: x86

--- a/manifests/g/Google/AndroidStudio/2023.1.1.27/Google.AndroidStudio.installer.yaml
+++ b/manifests/g/Google/AndroidStudio/2023.1.1.27/Google.AndroidStudio.installer.yaml
@@ -12,7 +12,7 @@ InstallerSuccessCodes:
 - 1223
 UpgradeBehavior: install
 AppsAndFeaturesEntries:
-- DisplayVersion: "2023.1"
+- DisplayVersion: "2023.1.1"
   ProductCode: Android Studio
 Installers:
 - Architecture: x86

--- a/manifests/g/Google/AndroidStudio/2023.1.1.27/Google.AndroidStudio.installer.yaml
+++ b/manifests/g/Google/AndroidStudio/2023.1.1.27/Google.AndroidStudio.installer.yaml
@@ -15,7 +15,7 @@ AppsAndFeaturesEntries:
 - DisplayVersion: "2023.1.1.27"
   ProductCode: Android Studio
 Installers:
-- Architecture: x86
+- Architecture: x64
   InstallerUrl: https://redirector.gvt1.com/edgedl/android/studio/install/2023.1.1.27/android-studio-2023.1.1.27-windows.exe
   InstallerSha256: 266F89313953E9EE01677A8B038588B1AACC4B6F687BC1046A5D65F08EAC9F2F
 ManifestType: installer

